ddl/placement/const.go

```diff
@@ -33,6 +33,14 @@ const (
 	RuleIndexIndex
 )
 
-// DCLabelKey indicates the key of label which represents the dc for Store.
-// FIXME: currently we assumes "zone" is the dcLabel key in Store
-const DCLabelKey = "zone"
+const (
+	// DCLabelKey indicates the key of label which represents the dc for Store.
+	// FIXME: currently we assumes "zone" is the dcLabel key in Store
+	DCLabelKey = "zone"
+	// EngineLabelKey is the label that indicates the backend of store instance:
+	// tikv or tiflash. TiFlash instance will contain a label of 'engine: tiflash'.
+	EngineLabelKey = "engine"
+	// EngineLabelTiFlash is the label value, which a TiFlash instance will have with
+	// a label key of EngineLabelKey.
+	EngineLabelTiFlash = "tiflash"
+)
```

Comment:
I mean, some other places also use "engine" and "tiflash", especially the code relevant to `set tiflash replica`. This would make duplicates.

Reply:
Done